app/src/main/java/com/github/se/studybuddies/ui/topics/TopicResources.kt

```diff
@@ -140,6 +181,22 @@ fun TopicResources(
       }
 }
 
+@Composable
+fun ResourceImage(image: Uri) {
+  Log.d("Topics", "current image is $image")
+  Row(
+      modifier = Modifier.fillMaxWidth().padding(8.dp),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.Center) {
+        Image(
+            painter = rememberAsyncImagePainter(image.toString()),
+            contentDescription = stringResource(R.string.image_resource),
+            modifier =
+                Modifier.size(200.dp).clip(RoundedCornerShape(20.dp)).testTag("chat_message_image"),
```

Comment:
is this the correct test tag or just a reused one ?

Reply:
yep, a reused one! I missed it but it's removed now